[2022-02-03 14:47:19,153] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: FHV_data_ingestion_gcs_dag.local_to_gcs_task scheduled__2019-10-01T00:00:00+00:00 [queued]>
[2022-02-03 14:47:19,167] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: FHV_data_ingestion_gcs_dag.local_to_gcs_task scheduled__2019-10-01T00:00:00+00:00 [queued]>
[2022-02-03 14:47:19,168] {taskinstance.py:1238} INFO - 
--------------------------------------------------------------------------------
[2022-02-03 14:47:19,169] {taskinstance.py:1239} INFO - Starting attempt 1 of 1
[2022-02-03 14:47:19,169] {taskinstance.py:1240} INFO - 
--------------------------------------------------------------------------------
[2022-02-03 14:47:19,181] {taskinstance.py:1259} INFO - Executing <Task(PythonOperator): local_to_gcs_task> on 2019-10-01 00:00:00+00:00
[2022-02-03 14:47:19,185] {standard_task_runner.py:52} INFO - Started process 1372 to run task
[2022-02-03 14:47:19,189] {standard_task_runner.py:76} INFO - Running: ['***', 'tasks', 'run', 'FHV_data_ingestion_gcs_dag', 'local_to_gcs_task', 'scheduled__2019-10-01T00:00:00+00:00', '--job-id', '457', '--raw', '--subdir', 'DAGS_FOLDER/FHV_injection_dag.py', '--cfg-path', '/tmp/tmpthocn2sr', '--error-file', '/tmp/tmphdlin9k1']
[2022-02-03 14:47:19,191] {standard_task_runner.py:77} INFO - Job 457: Subtask local_to_gcs_task
[2022-02-03 14:47:19,248] {logging_mixin.py:109} INFO - Running <TaskInstance: FHV_data_ingestion_gcs_dag.local_to_gcs_task scheduled__2019-10-01T00:00:00+00:00 [running]> on host 700aeb16bae0
[2022-02-03 14:47:19,283] {warnings.py:110} WARNING - /home/***/.local/lib/python3.7/site-packages/***/utils/context.py:152: AirflowContextDeprecationWarning: Accessing 'execution_date' from the template is deprecated and will be removed in a future version. Please use 'data_interval_start' or 'logical_date' instead.
  warnings.warn(_create_deprecation_warning(key, self._deprecation_replacements[key]))

[2022-02-03 14:47:19,306] {taskinstance.py:1426} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=FHV_data_ingestion_gcs_dag
AIRFLOW_CTX_TASK_ID=local_to_gcs_task
AIRFLOW_CTX_EXECUTION_DATE=2019-10-01T00:00:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2019-10-01T00:00:00+00:00
[2022-02-03 14:47:21,481] {python.py:175} INFO - Done. Returned value was: None
[2022-02-03 14:47:21,497] {taskinstance.py:1277} INFO - Marking task as SUCCESS. dag_id=FHV_data_ingestion_gcs_dag, task_id=local_to_gcs_task, execution_date=20191001T000000, start_date=20220203T144719, end_date=20220203T144721
[2022-02-03 14:47:21,562] {local_task_job.py:154} INFO - Task exited with return code 0
[2022-02-03 14:47:21,600] {local_task_job.py:264} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2022-02-03 15:12:07,558] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: FHV_data_ingestion_gcs_dag.local_to_gcs_task scheduled__2019-10-01T00:00:00+00:00 [queued]>
[2022-02-03 15:12:07,573] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: FHV_data_ingestion_gcs_dag.local_to_gcs_task scheduled__2019-10-01T00:00:00+00:00 [queued]>
[2022-02-03 15:12:07,574] {taskinstance.py:1238} INFO - 
--------------------------------------------------------------------------------
[2022-02-03 15:12:07,575] {taskinstance.py:1239} INFO - Starting attempt 1 of 1
[2022-02-03 15:12:07,576] {taskinstance.py:1240} INFO - 
--------------------------------------------------------------------------------
[2022-02-03 15:12:07,593] {taskinstance.py:1259} INFO - Executing <Task(PythonOperator): local_to_gcs_task> on 2019-10-01 00:00:00+00:00
[2022-02-03 15:12:07,599] {standard_task_runner.py:52} INFO - Started process 2970 to run task
[2022-02-03 15:12:07,608] {standard_task_runner.py:76} INFO - Running: ['***', 'tasks', 'run', 'FHV_data_ingestion_gcs_dag', 'local_to_gcs_task', 'scheduled__2019-10-01T00:00:00+00:00', '--job-id', '559', '--raw', '--subdir', 'DAGS_FOLDER/FHV_injection_dag.py', '--cfg-path', '/tmp/tmpicw5q6yk', '--error-file', '/tmp/tmpoo2bbtl5']
[2022-02-03 15:12:07,611] {standard_task_runner.py:77} INFO - Job 559: Subtask local_to_gcs_task
[2022-02-03 15:12:07,699] {logging_mixin.py:109} INFO - Running <TaskInstance: FHV_data_ingestion_gcs_dag.local_to_gcs_task scheduled__2019-10-01T00:00:00+00:00 [running]> on host 700aeb16bae0
[2022-02-03 15:12:07,751] {warnings.py:110} WARNING - /home/***/.local/lib/python3.7/site-packages/***/utils/context.py:152: AirflowContextDeprecationWarning: Accessing 'execution_date' from the template is deprecated and will be removed in a future version. Please use 'data_interval_start' or 'logical_date' instead.
  warnings.warn(_create_deprecation_warning(key, self._deprecation_replacements[key]))

[2022-02-03 15:12:07,779] {taskinstance.py:1426} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=FHV_data_ingestion_gcs_dag
AIRFLOW_CTX_TASK_ID=local_to_gcs_task
AIRFLOW_CTX_EXECUTION_DATE=2019-10-01T00:00:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2019-10-01T00:00:00+00:00
[2022-02-03 15:12:10,296] {python.py:175} INFO - Done. Returned value was: None
[2022-02-03 15:12:10,312] {taskinstance.py:1277} INFO - Marking task as SUCCESS. dag_id=FHV_data_ingestion_gcs_dag, task_id=local_to_gcs_task, execution_date=20191001T000000, start_date=20220203T151207, end_date=20220203T151210
[2022-02-03 15:12:10,385] {local_task_job.py:154} INFO - Task exited with return code 0
[2022-02-03 15:12:10,421] {local_task_job.py:264} INFO - 0 downstream tasks scheduled from follow-on schedule check
